dispersion_protocol/messages.md

```diff
@@ -0,0 +1,183 @@
+| vx                              | float      |  m/s  | invalid:NaN                                                                                 | X Speed in NED (North, East, Down). NAN if unknown.                                                                                                         |
+| vy                              | float      |  m/s  | invalid:NaN                                                                                 | Y Speed in NED (North, East, Down). NAN if unknown.                                                                                                         |
+| vz                              | float      |  m/s  | invalid:NaN                                                                                 | Z Speed in NED (North, East, Down). NAN if unknown.                                                                                                         |
+| v_estimated_delay_us            | uint32_t   |  us   | invalid:0                                                                                   | Estimated delay of the speed data. 0 if unknown.                                                                                                            |
+| feed_forward_angular_velocity_z | float      | rad/s | invalid:NaN                                                                                 | Feed forward Z component of angular velocity (positive: yawing to the right). NaN to be ignored. This is to indicate if the autopilot is actively yawing.   |
+| estimator_status                | uint16_t   |       | [ESTIMATOR_STATUS_FLAGS](https://mavlink.io/en/messages/common.html#ESTIMATOR_STATUS_FLAGS) | Bitmap indicating which estimator outputs are valid.                                                                                                        |
+| landed_state                    | uint8_t    |       | invalid:MAV_LANDED_STATE_UNDEFINED MAV_LANDED_STATE                                         | The landed state. Is set to [MAV_LANDED_STATE_UNDEFINED](https://mavlink.io/en/messages/common.html#MAV_LANDED_STATE_UNDEFINED) if landed state is unknown. |
+| angular_velocity_z              | float      | rad/s | invalid:NaN                                                                                 | Z component of angular velocity in NED (North, East, Down). NaN if unknown.                                                                                 |
+
+> [!QUESTION]
+> This table is an exact clone of AUTOPILOT_STATE_FOR_GIMBAL_DEVICE because it felt like a good representation of the information a dispersion controller might need to deliver quality corrections. It at a minimum needs velocity. Can we just use something like GLOBAL_POS_INT or the GIMBAL message? I dont love the redundancy.
```

Comment:
> Can we just use something like GLOBAL_POS_INT or the GIMBAL message? I dont love the redundancy.

Yes, and that is likely to be preferred. Your protocol documentation would indicate a dependency on the messages you need for the whole system to work.